bin/package/installation/initd.sh

```diff
@@ -0,0 +1,220 @@
+#!/bin/bash
+### BEGIN INIT INFO
+# Provides:          mysterium-node
+# Required-Start:    $all
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start the Mysterium VPN node process
+### END INIT INFO
+
+# If you modify this, please make sure to also edit systemd.service
+
+OS_DIR_BIN="/usr/bin"
+OS_DIR_CONFIG="/etc/mysterium-node"
```

Comment:
Where should be config|logs|data directories: `/etc/myst` or `/etc/mysterium-node` ?

Reply:
Well, my idea was that this service is starting mysterium exit node, so all configs and logs should be in the `mysterium-node` directories and I left it as it was before.